nucliadb/src/nucliadb/search/search/query.py

```diff
@@ -463,7 +483,8 @@ async def parse_vector_search(self, request: nodereader_pb2.SearchRequest) -> bo
                         else:
                             incomplete = True
                     else:
-                        query_vector = query_info.sentence.data
+                        for query_vector in query_info.sentence.vectors.values():
```

Comment:
maybe it's better to use popitem? I know it shouldn't but what if for some reason vectors is empty?

Reply:
I'd prefer not to modify the `QueryInfo` payload, but I can check for empty vector